pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -237,12 +300,15 @@ func (b *AccessLogBuilder) setHTTPAccessLog(push *model.PushContext, proxy *mode
 		if mesh.EnableEnvoyAccessLogService {
 			connectionManager.AccessLog = append(connectionManager.AccessLog, b.httpGrpcAccessLog)
 		}
+
+		b.store.Add(k, connectionManager.AccessLog)
```

Comment:
should we use the `addAccessLog` function instead of adding directly here so that we get locks?

Reply:
yes, it's my mistake